admin/tsconfig.json

```diff
@@ -2,9 +2,6 @@
     "include": ["src/**/*.ts", "src/**/*.tsx"],
     "compilerOptions": {
         "baseUrl": "./src",
-        "paths": {
-            "@/*": ["*"]
-        },
```

Comment:
This import path was added to make absolute imports work in Typescript, like `import {get} from "@/gateway";` instead of doing relative imports `import {get} from "../../../gateway";`. Relative imports are IMO much harder to read and recognize, since they are different depending on the path of each file.

Reply:
But we don't use any imports of the form "@/..." in the codebase right now?

Reply:
No, that is true. I added it a bit prematurely because I wanted to updated the include paths later on.

Reply:
Will do